graalpy-install-package/src/main/java/com/oracle/example/graalpy/PyfigletFrame.java

```diff
@@ -74,6 +76,9 @@ public class PyfigletFrame extends JFrame {
    * Creates new PyfigletFrame
    */
   public PyfigletFrame(PyfigletProxy proxy) {
+    if (System.getProperty("java.home") == null || System.getProperty("java.home").isEmpty()) {
```

Comment:
I think this can be simplified to:
```suggestion
    if (System.getProperty("java.home", "").isEmpty()) {
```

Are you sure this is really necessary? When would `java.home` not be set?

Reply:
It's not set when you are running the generated native image